students/NatalieRodriguez/Lesson02/Fizz_Buzz.py

```diff
@@ -0,0 +1,15 @@
+for i in range(0,101):
```

Comment:
note that zero is the default start for `range` so you can either do `range(1, 101)` to start at 1 or `range(101)` to start at zero